_posts/2015-04-28-how-I-found-the-cheapest-place-to-live-in-austin.md

```diff
@@ -0,0 +1,274 @@
+table with a DJ set, kitchen, bathroom and the master bedroom. And no WiFi. 
+Crap. Alright, we'll get by. I ended up working out of a coffee shop, and
+cooking eggs in the microwave inside a glass.
+
+It was a really fun experience. He showed me around, drove me places when I
+needed to, and was really eager to give the NomadPad a try. He wanted to 
+establish a network of houses throughout the world that would enable you to
+travel from place to place without having to wonder about leasing fees or other
+rental headaches. We kept talking about it, and in fact one of the reasons he 
+agreed to host me was that he wanted me to be a member of the first house. I was
+totally in. He also shared more of his adventures through Asia and how he became 
+a DJ, later to be chased by the Thai mafia. I'll leave that story for another 
+post.
+
+The original plan was for me to stay during January at Randall's. The NomadPad 
+was scheduled to start in February, and I was ready to move in as soon as we
+figured it all out. However, we soon discovered a flaw in our plans: SXSW.
+
+Now let me backtrack a bit and tell you about SXSW. South by Southwest (SXSW) is
+a set of film, interactive, and music festivals and conferences that take place
+each year in mid-March in Austin. The best way to describe it is this:
+Imagine the whole city being crowded as if it were a concert. That's how many
+people attend the event. Finding a place to stay is incredibly difficult during
+that week, and prices shoot up from $70/night up to $1000/night in some cases.
+It's such a huge deal that some native Austinites leave for a vacation, which is
+paid for by the money they receive from renting their homes.
+
+Now, because Randall still didn't have a house secured for the NomadPad. And he
+needed good prices to make the NomadPad profitable, i.e. find a long term lease.  
+That's why SXSW became a problem.
+
+Instead, Randall decided to change plans and open up the NomadPad in April,
+which left me in a rather awkward situation. I didn't know if I were to be
+staying at his place during the three months, or if I had to find a place in the
+meantime, after January was over. And before I began asking myself about this,
+Randall told me he would be taking a trip to Dallas because of business reasons 
+in a couple days, and that, although he knew it was kinda difficult to find a
+place with such a short notice, he half-expected me to have something figured
+out by the time he came back, about four days later. So there was my answer. Not
+even 'til the end of January.
+
+The search for Home #3 had begun. More like a week had passed by.
+
+# Jen and Tori
+
+Two days before Randall came back, I start to become increasingly worried about 
+finding a place within my budget. I had never used Craigslist or anything else;
+it was my first time looking for a house with time pressure. I figured I'd just 
+find a short-term deal so that I can buy myself some time until I find a good 
```

Comment:
I figured I'd just find a short-term deal so that I _could_ buy myself some time until I _found_ a good place.